pyafq_cli.json

```diff
@@ -0,0 +1,501 @@
+    },
+    {
+        "id": "segmentation_group",
+        "name": "Segmentation Group",
+        "description": "Parameters related to segmentation.",
+        "members": ["nb_points", "segmentation_seg_algo", "progressive",
+                    "greater_than", "rm_small_clusters", "model_clust_thr",
+                    "reduction_thr", "refine", "pruning_thr", "b0_threshold",
+                    "prob_threshold", "rng", "segmentation_return_idx",
+                    "filter_by_endpoints", "dist_to_aal"]
+    },
+    {
+        "id": "cleaning_group",
+        "name": "Cleaning Group",
+        "description": "Parameters related to cleaning.",
+        "members": ["n_points", "clean_rounds", "distance_threshold",
+                    "length_threshold", "min_sl", "stat", "cleaning_return_idx"]
+    }
+    ],
+    "inputs": [
+    {
+        "id": "config_file",
+        "name": "Configuration file",
+        "type": "String",
+        "value-key": "[CONFIG_FILE]",
+        "optional": false
+    },
+    {
+        "id": "dmriprep_path",
+        "name": "dmriprep path",
+        "type": "String",
+        "value-key": "[DMRIPREP_PATH]",
+        "optional": false
+    },
+    {
+        "id": "bundles",
+        "name": "bundles",
+        "type": "String",
+        "list": true,
+        "min-list-entries": 1,
+        "list-separator": "', '",
+        "value-key": "[BUNDLES]",
+        "value-choices": ["ATR", "CGC", "CST", "HCC", "IFO", "ILF",
+                          "SLF", "ARC", "UNC", "FA", "FP"],
+        "default-value": ["ATR", "CGC", "CST", "HCC", "IFO", "ILF",
+                          "SLF", "ARC", "UNC", "FA", "FP"],
+        "optional": true
+    },
+    {
+        "id": "bundles_seg_algo",
+        "name": "bubndles seg algo",
+        "type": "String",
+        "value-key": "[BUNDLES_SEG_ALGO]",
+        "value-choices": ["AFQ"],
+        "default-value": "AFQ",
+        "optional": true
+    },
+    {
+        "id": "scalars_model",
+        "name": "scalars model",
+        "type": "String",
+        "value-key": "[SCALARS_MODEL]",
+        "value-choices": ["DTI"],
+        "default-value": "DTI",
+        "optional": true
+    },
+    {
+        "id": "scalars",
+        "name": "scalars",
+        "type": "String",
+        "list": true,
+        "min-list-entries": 1,
+        "list-separator": "', '",
+        "value-key": "[SCALARS]",
+        "value-choices": ["dti_fa", "dti_md"],
+        "default-value": ["dti_fa", "dti_md"],
+        "optional": true
+    },
+    {
+        "id": "directions",
+        "name": "directions",
+        "type": "String",
+        "value-key": "[DIRECTIONS]",
+        "value-choices": ["det", "prob"],
+        "default-value": "det",
+        "optional": true
+    },
+    {
+        "id": "max_angle",
+        "name": "max angle",
+        "type": "Number",
+        "value-key": "[MAX_ANGLE]",
+        "minimum": 0.0,
+        "maximum": 180.0,
```

Comment:
Are `[0.0, 180.0]` correct extreme values for the angle?

Reply:
I believe so.